### PR TITLE
First draft of staggered DiD BB

### PR DESCRIPTION
**Pull request by srosh2000**
Here's a first draft! I have put it under regressions section as a placeholder till we figure out the positioning of the causal inference content. Looking forward to your feedback! :)

### CODE REVIEW

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+weight: 2
+author: "Roshini Sudhaharan"
+authorlink: "https://nl.linkedin.com/in/roshinisudhaharan"
+aliases:
+  - /staggered-did
+---
+
+# Overview
+Difference-in-Difference (DiD) analysis serves as a valuable tool for evaluating the causal impact of interventions in observational studies. It involves comparing changes in outcomes between a treatment group and a control group before and after the introduction of an intervention. While the traditional DiD framework typically revolves around two distinct groups and two specific time periods, real-world applications often present more intricate scenarios with multiple time intervals and varying treatment timings.
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:15]: cite your other building block here?

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+Several factors contribute to this limitation, including the potential for the treatment effect to fluctuate over time or among different groups due to factors such as learning curves, adaptation, or changing external conditions. Additionally, some units may exhibit immediate responses to treatment, while others might require more time to react. The conventional approach's one-size-fits-all methodology may not effectively account for these variations.
+
+More specifically, the treatment effect estimate in the traditional DiD approach is essentially a weighted average of treatment effects. These weights can sometimes be negative, even if the overall treatment effect is positive. Consequently, this can lead to misleading estimates of the treatment effect, where the average effect appears negative, despite the actual positive impact of the intervention.
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:36]: mention that "additional resources/see also" section contains more details

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+## An Illustrative Example: Steps for Staggered DiD Estimation
+
+In this illustrative example, we investigate the impact of the Questions and Answers (Q&A) feature on ratings and reviews within online platforms. Specifically, we analyze data from two prominent platforms: Goodreads and Amazon. Goodreads serves as the treatment group, as it offers users the ability to rate and review books while also providing the Q&A feature. In contrast, Amazon serves as the control group because it enables users to rate and review products but lacks the Q&A feature for books.
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:40]: mention that using amazon as the control group means that we assume no spillovers between the 2, and no time varying differences. might to good to add this as a tip or note, with the general lesson that the considerations for choice of control group still remain the same regardless of staggered vs regular DiD

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+This scenario presents a staggered treatment setup, where various books receive questions at different points in time. This staggered treatment timing introduces variations that would yield biased estimates when using the conventional DiD framework. To address this issue, we employ a modern DiD technique proposed by [Callaway and Sant Anna (2021)](https://www.sciencedirect.com/science/article/pii/S0304407620303948), implemented using the `did` [package](https://bcallaway11.github.io/did/articles/did-basics.html).
+
+This technique offers two primary advantages:
+
+1. **Enhanced Control Group Selection:** It provides greater flexibility in choosing control groups. Specifically, it allows us to utilize "not-yet treated units" as controls, in addition to "never-treated" units. In our context, this means that a book receiving its first question in July 2017 can serve as a control for a book that received its first question in May 2017. The "never-treated" units comprise books on Amazon that never receive questions.
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:46]: you introduce the setup by saying we'll use amazon as control, so the not-yet-treated comes as a bit of a surprise. clarify upfront that we will use 2 different control groups, and then go through them one by one

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+- Identifying assumptions for staggered DiD include the persistence of treated units and (conditional) parallel trends.
+
+- Staggered DiD can be estimated in R using the `did` package.
+
+- `att_gt()` is used to estimate group-time average treatment effect.
+
+- `aggte()` is used to *aggregate* group-time average treatment effects.
+
+- `ggdid()` is used to visualise the treatment effects.
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:207]: perhaps prettify the tables using modelsummary, etable or similar?

File: content/building-blocks/analyze-data/regressions/staggered-did.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+```
+{{% /codeblock %}}
+
+Notice how 12 months prior, there are essentially no differences between treatment and control group units which provides some support for the parallel trends assumption. But, post Q&A, the ratings go down.
+
+<p align = "center">
+<img src = "../images/ggdid-notyettreated.png" width="700">
+<figcaption> Event Study Plot </figcaption>
+</p>
+
```
Comment (shrabasteebanerjee) [inline content/building-blocks/analyze-data/regressions/staggered-did.md:190]: do you also want to add never treated results, given you talk about the method?

**Review by shrabasteebanerjee — CHANGES_REQUESTED** [review]
added my comments!

**Comment by hannesdatta** [comment]
@srosh2000 - could you please take a look at @shrabasteebanerjee comments and implement them when you find the time? note also i rewrote the beginning a bit - it was very wordy. Try to shorten/be concise in your writing.

**Comment by srosh2000** [comment]
@hannesdatta Noted! Yes it's on my to do list. Will finalise it this coming week

**Comment by srosh2000** [comment]
Made a new PR incorporating the changes. Made a new PR as I had to relocate this file. Closing this PR for now,

**Pull request closed without merging**